catalogs/catalog-lakehouse-paimon/src/main/java/com/datastrato/gravitino/catalog/lakehouse/paimon/PaimonSchema.java

```diff
@@ -4,9 +4,14 @@
  */
 package com.datastrato.gravitino.catalog.lakehouse.paimon;
 
+import static com.datastrato.gravitino.meta.AuditInfo.EMPTY;
+
 import com.datastrato.gravitino.Schema;
 import com.datastrato.gravitino.connector.BaseSchema;
+import java.util.Map;
+import java.util.Optional;
 import lombok.ToString;
+import org.apache.paimon.utils.Pair;
```

Comment:
use `org.apache.commons.lang3.tuple.Pair` ?

Reply:
ok for me.